mne/io/proj.py

```diff
@@ -112,6 +112,10 @@ def add_eeg_average_proj(self):
             # Don't set as active, since we haven't applied it
             eeg_proj = make_eeg_average_ref_proj(self.info, activate=False)
             self.add_proj(eeg_proj)
+        elif self.info['custom_ref_applied']:
+            warn('A custom reference was already applied to the data. No '
+                 'average reference was added.')
```

Comment:
Just re-commenting here since my last one was buried -- shouldn't it be an error if someone tries to add an avg proj but we won't do it? Seems safer than a warning